tools/recetox_aplcms/macros.xml

```diff
@@ -146,6 +145,25 @@
                    help="The minimum number of raw data points to be considered as a true feature." />
         </section>
     </xml>
+    <xml name="multibatch_processing">
+        <section name="multibatch_processing" title="Multibatch processing">
+            <param name="min_within_batch_prop_detect" type="float" min="0" max="1" value="0.1"
+                   label="minimum_batchwise_prop_detect"
+                   help="The minimum proportion of samples in a batch that must have a given feature. The features that are less abundant than the value won't be extracted." />

```

Comment:
The term `abundant` here is confusing. I'd suggest
`The minimum detection frequency of a feature for it to be included in the aligned feature table`.

Reply:
Changed description in 3e752f9.